proto/projects/project.proto

```diff
@@ -18,15 +18,18 @@ message Project {
 }
 
 message ProjectKey {
+    string key = 1; // the address of the project key
+
+    reserved 2;
     reserved 3;
-    string  key = 1; // the address of the project key
-    enum KEY_TYPE{ // bitmap, must only be power of 2
-        NONE = 0;
-        ADMIN = 1;
-        DEVELOPER = 2; 
-    }    
-
-    repeated KEY_TYPE types = 2 [(gogoproto.nullable) = false]; // the key type, determines the privilages of the key
+
+    enum Type {
+        NONE = 0x0;
+        ADMIN = 0x1;
+        DEVELOPER = 0x2;
+    }
+
+	uint32 kinds = 4;
```

Comment:
can we keep the name "types"?

Reply:
using "kinds" allows to have a convenient variable named "kind" (whereas "type" is a reserved word).